recipes/readline/all/conanfile.py

```diff
@@ -66,6 +66,10 @@ def _configure_autotools(self):
             configure_args.append("bash_cv_wcwidth_broken=yes")
 
         self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+        if self.settings.os == "Macos" and self.settings.arch == "armv8":
```

Comment:
```suggestion
        if self.settings.arch == "armv8":
```

Reply:
or maybe tools.is_apple_os(self.settings.os), but for sure not only Macos.

Reply:
I can't test other configurations right now, it's out of scope of the PR

Reply:
`armv8` is not only 64-bit architecture from `settings.yml`, `armv8.3` is another one. Probably check should be for prefix `armv8` and not `armv8_32`

Reply:
I think SSE4's initial code is correct, though I would check for the compiler (`self.settings.compiler == "apple-clang"`) instead.
The added link flags look compiler/linker specific.